css/style.scss

```diff
@@ -0,0 +1,52 @@
+@charset 'utf-8';
+// import settings first
+@import 'settings/colors';
+@import 'settings/sizings';
+// then import components
+@import 'components/btn';
+@import 'components/form-questions';
+@import 'components/modal';
+@import 'components/table/page-table';
+@import 'components/table/question-table';
+
+
+.wp-menu-image {
```

Comment:
Try not having any rule in the main file. The main file is usually used just for combining everything together.